packages/tokens/src/tokens/core/fonts.tokens.json

```diff
@@ -0,0 +1,124 @@
+{
+    "font": {
+        "family": {
+            "primary" : {
+                "$type": "string",
+                "$value": "ABC Favorit"
+            },
+            "secondary" : {
+                "$type": "string",
+                "$value": "Inter"
+            },
+            "tertiary" : {
+                "$type": "string",
+                "$value": "ABC Favorit Mono"
+            }
+        },
+        "size": {
+            "120": {
+                "$type": "number",
+                "$value": "12"
+            },
+            "140": {
+                "$type": "number",
+                "$value": "14"
+            },
+            "160": {
+                "$type": "number",
+                "$value": "16"
+            },
+            "180": {
+                "$type": "number",
+                "$value": "18"
+            },
+            "200": {
+                "$type": "number",
+                "$value": "20"
+            },
+            "240": {
+                "$type": "number",
+                "$value": "24"
+            },
+            "280": {
+                "$type": "number",
+                "$value": "28"
+            },
+            "320": {
+                "$type": "number",
+                "$value": "32"
+            },
+            "360": {
+                "$type": "number",
+                "$value": "36"
+            },
+            "480": {
+                "$type": "number",
+                "$value": "48"
+            }
+        },
+        "weight": {
+            "690": {
+                "$type": "number",
+                "$value": "690"
+            },
+            "680": {
+                "$type": "number",
+                "$value": "680"
+            },
+            "590": {
+                "$type": "number",
+                "$value": "590"
+            },
+            "580": {
+                "$type": "number",
+                "$value": "580"
+            },
+            "505": {
+                "$type": "number",
+                "$value": "505"
+            },
+            "410": {
+                "$type": "number",
+                "$value": "410"
+            },
+            "400": {
+                "$type": "number",
+                "$value": "400"
+            }
+        }
+    },
+    "line-height": {
+        "112-5": {
```

Comment:
In some occurences for technical reasons I use a line height of 1 (Buttons / Tag, to prevent an image an inline image to take more space than its own height, although there is other tricks to solve this common issue) situations where the text will never be on two lines, I am wondering if we should add it here, that would be available if need be, and prevent having to use a literal : 1.

Reply:
I think it's better to add it the day the case arises.